assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -182,14 +182,22 @@ const drawRadioButtonRow = function(container, parameter, store) {
 * @param {D3 selection} parameter - Contains details about the current parameter code
 */
 const drawWaterAlertRow = function(container, siteno, parameter) {
+    // WaterAlert doesn't accept the calculated parameter for Fahrenheit (such as 00010F), so we need to adjust the
+    // parameter code back to the Celsius version (such as 00010).
+    const waterAlertURL = function() {
```

Comment:
I'm wondering if we should include the waterAlertURL in the data returned from the selector? That would keep all of the Fahrenheit/Celsius code in a selector which is easier to test.

Reply:
I like that idea, but how do we get the siteno in the selector?

Reply:
I went back and forth on this. I tried passing a function with the URL that accepted a site number, but that didn't seem great. Then, in looking more closely, I realized that the logic for the adjusting the calculated parameter code already was in the parameter-data and just wasn't making it to the right spot. So I compromised and used that to shorten the logic that creates the URL.